bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/action/DynamicsActions.java

```diff
@@ -36,7 +36,7 @@
  * @author Andrew Fiddian-Green - Initial contribution
  */
 @Component(scope = ServiceScope.PROTOTYPE, service = DynamicsActions.class)
-@ThingActionsScope(name = "hue")
```

Comment:
Please do not rename this scope. This is for API v2, which we assume is the most used version, so to avoid breaking changes for the majority of users, we should not rename this.

Reply:
Done